ext/opcache/jit/ir/ir_x86.dasc

```diff
@@ -9221,6 +9263,54 @@ static void ir_emit_tailcall(ir_ctx *ctx, ir_ref def, ir_insn *insn)
 		return;
 	}
 
+	/* Move op2 to a scratch register before epilogue if it's in
+	 * used_preserved_regs, because it will be overridden. */
```

Comment:
This fixes a bug where TAILCALL() op2 would be overridden by epilogue before the call.

I will submit this as a proper PR separately.

Reply:
https://github.com/dstogov/ir/pull/115